pandora-client-web/src/components/releationships/relationships.tsx

```diff
@@ -25,10 +25,10 @@ export function Relationships() {
 	return (
 		<div className='relationships'>
 			<UrlTabContainer>
-				<UrlTab name={ <RelationshipHeader type='friend' /> } urlChunk='friends'>
+				<UrlTab name={ <RelationshipHeader type='friend' /> } urlChunk='list'>
```

Comment:
shouldn't this change be: `frients` -> `contacts`? instead of the `list`

Reply:
Well, the URL would be `contacts/contacts` then, which is a bit silly. It could also be changed to `contacts/main` maybe.